### ISSUE TITLE
MultiLang support

### ISSUE THREAD

**Issue by foxpcteam** [squizlabs/HTML_CodeSniffer#190]
Hi,
any plan to add i18n?

**Comment by TheJaredWilcurt** [comment]
It's a little extra work, but not that hard to set up. I did it for [Scout-App](http://scout-app.io/#cultures).

How it works for us:

1. Create a public [Google Sheet](https://docs.google.com/spreadsheets/d/16FtHGThz0-CBz_rf_9QO4zOV7oNmNk2BlwmtvKuzAhs/edit?usp=sharing)
2. Add a `KEYWORDS` column that will be used in your codebase and swapped out with the related translation.
3. Add a `CONTEXT` column that explains/describes the context of where/how this text is used.
4. Add a `en` column for the actual text you want to be displayed.

Then let others go though and translate all those keys for you.

I ended up creating my own Node script that I can run to download the CSV, process it, and output a dictionary.json file for my app to use.

* https://github.com/scout-app/scout-app/blob/master/cultures.js

I duplicate and lock down the sheet when it's time for a release, so there is a record kept. And when I need to add new rows of text, I just Google translate it and add a note of that so others can improve the translation if needed.

There are websites out there specifically for i18n, but most cost money or are hard to use. Google sheets is a really low barrier of entry.
